### ISSUE TITLE
Errors/warnings should go to stderr

### ISSUE THREAD

**Issue by wknapik** [kubernetes/kops#2861]
[This](https://github.com/kubernetes/kops/blob/8585f00717fb7d83efa176e9de205a9f74c0d3a6/cmd/kops/edit_cluster.go#L157)  is not funny.

Consider the following code:
```
EDITOR=cat kops edit cluster|something|kops replace -f /dev/stdin
```

You end up with "Edit cancelled, no changes made." in the middle of your config.

The same is done in [other places](https://github.com/kubernetes/kops/search?utf8=%E2%9C%93&q=Edit+cancelled%2C+no+changes+made.&type=).

PS. Also, `kops replace` doesn't seem to want to read from stdin without options, nor with `-f -` (the latter causes `error: error reading file "-": open -: no such file or directory`).

**Comment by wknapik** [comment]
Created #2862 for the "PS." part.

**Comment by chrislovecnm** [comment]
Sdtin - is a known issue that is a open product enhancement request.

The error code path needs to be handled by the root.go or main.go error handler. Forget where it is.

**Comment by wknapik** [comment]
Appears to be resolved in 1.9, or earlier.
/close
